db/seeds.rb

```diff
@@ -34,6 +34,11 @@
   username: 'Reader', password: 'reader', email: "dummy7@example.com", role_id: Permissible::READONLY,
 )
 
+User.create!(
+  username: 'red', password: 'red', email: "dummy8@example.com", role_id: Permissible::MOD,
+  layout: 'starrydark', ignore_unread_daily_report: true, created_at: "2024-11-30 15:34:00",
```

Comment:
I'm not sure you want the created_at in there? otherwise looks good tho

Reply:
Seems like the other seeds (except reader user) have created_at, interestingly 🤔

Reply:
huh, so they do! i wonder why we did that. nvm then

Reply:
no idea tbh, it's weird. cool!